src/Driver/IBMDB2/Statement.php

```diff
@@ -211,8 +215,23 @@ private function createTemporaryFile()
      */
     private function copyStreamToStream($source, $target): void
     {
+        $resetTo = false;
+        try {
+            if (stream_get_meta_data($source)['seekable']) {
+                $resetTo = ftell($source);
+            }
+        } catch (Throwable $e) {
+            // Swallow
+        }
+
         if (@stream_copy_to_stream($source, $target) === false) {
             throw CannotCopyStreamToStream::new(error_get_last());
```

Comment:
`stream_copy_to_stream()` might have moved the stream pointer before failing. We might need to rewind it in that situation too.

Reply:
Possibly, but at this point you're already in an exception state. The reset could potentially also trigger the same underlying exception that caused the stream_copy_to_stream failure, which would mask the original error, so we would want to swallow any fseek failure and throw the original.

I'm open to the reset if you want to proceed.

Reply:
> The reset could potentially also trigger the same underlying exception that caused the stream_copy_to_stream failure, which would mask the original error, so we would want to swallow any fseek failure and throw the original.

Yes. We should try to `fseek()` and silence all errors it might produce.